docs/blog/write-the-docs-2018-kickoff.rst

```diff
@@ -0,0 +1,77 @@
+.. post:: Jan 20, 2018
+   :tags: community updates, core team
+   :author: Mikey Ariel
+
+Write the Docs 2018 Kick-Off
+============================
+
+Greetings, global documentarians!
+
+Mikey here, writing from the deep dark cold European winter, but looking forward to spring and summer, full of sunshine and docs!
+Following Eric's winter-warming `2017 Stats <http://www.writethedocs.org/blog/write-the-docs-2017-stats/>`_ post, I'd like to highlight some of the activities, changes, and experiments that you can expect in 2018.
+
+These efforts are a part of optimizing our work in supporting and growing the community, and the also aim to keep things fresh and exciting for our documentarians.
+
+We hope that you'll enjoy the shiny new things and of course, if you'd like to contribute to any of these efforts or have experiments that you'd like to pursue, please write to us at support@writethedocs.org and let us know!
```

Comment:
A little bit flowery language here `and of course, if you'd like to contribute to any of these efforts or have experiments that you'd like to pursue` -- could be a bit more concise while getting the point across: `We hope you'll enjoy the new things we're trying. If you'd like to contribute, please write..`